code/modules/projectiles/guns/misc/chem_gun.dm

```diff
@@ -1,8 +1,8 @@
 //his isn't a subtype of the syringe gun because the syringegun subtype is made to hold syringes
 //this is meant to hold reagents/obj/item/gun/syringe
 /obj/item/gun/chem
-	name = "reagent gun"
-	desc = "A Nanotrasen syringe gun, modified to automatically synthesise chemical darts, and instead hold reagents."
+	name = "Reagent Repeater"
+	desc = "A Nanotrasen smartdart repeater rifle, modified to automatically synthesize piercing."
```

Comment:
```suggestion
	desc = "A Nanotrasen smartdart repeater rifle, modified to automatically synthesize piercing darts."
```
i don't know if all these instances of a naked "piercing" is intentional but "piercing" isn't... really a noun? so I'm just assuming it's not